app/assets/v2/js/cart-ethereum-polygon.js

```diff
@@ -381,6 +382,14 @@ Vue.component('grantsCartEthereumPolygon', {
 
       let gasLimit = 0;
 
+      // fetch gas prices from polygon gas tracker
+      if (!this.polygon.gasPrices) {
+        const priceUrl = 'https://api.polygonscan.com/api?module=gastracker&action=gasoracle&apikey=I28K1DVQAWAISBSI146I71YQDBK6N1C9GJ';
```

Comment:
We might want to switch this out for a paid account or move this to the backend and periodically update it? This is going to get hammered if we call it every time a user hits the checkout.